src/api/Folder.js

```diff
@@ -160,7 +161,7 @@ class Folder extends Item {
             new FileAPI(this.options),
             new WebLinkAPI(this.options),
         );
-        this.itemCache = (this.itemCache || []).concat(flattened);
+        this.itemCache = union(this.itemCache, flattened);
```

Comment:
Yeah I can most likely revert this now, i think it was only an issue when I was previously fetching the folder again for the shared link info and it would duplicate the entries